src/ripple/app/tx/impl/details/NFTokenUtils.cpp

```diff
@@ -636,5 +674,139 @@ deleteTokenOffer(ApplyView& view, std::shared_ptr<SLE> const& offer)
     return true;
 }
 
+bool
+repairNFTokenDirectoryLinks(ApplyView& view, AccountID const& owner)
+{
+    // Assume that no repair is required.
```

Comment:
I'm seeing lots of comments of like this - a comment and then a line of code immediately below it that doesn't need the comment. I'll tag the ones I'd consider removing, but I'll stop saying "because it mirrors the line of code below it and the code is better without things that don't pull their weight".